src/pfe/portal/routes/projects/links.route.js

```diff
@@ -43,6 +43,7 @@ router.post('/api/v1/projects/:id/links', validateReq, checkProjectExists, async
 
     await project.createLink({
       projectID: targetProjectID,
+      projectName: targetProject.name,
```

Comment:
does this still work if the `targetProject` has been deleted (given that we verify the target project exists a few lines above) ?